windows-apps-src/publish/upload-app-packages.md

```diff
@@ -9,7 +9,7 @@ ms.localizationpriority: medium
 ---
 # Upload app packages
 
-The **Packages** page is where you upload all of the package files (.msix, .msixupload, .msixbundle, .appx, .appxupload, and/or .appxbundle) for the app that you're submitting. You can upload all your packages for the same app on this page, and when a customer downloads your app, the Store will automatically provide each customer with the package that works best for their device. After you upload your packages, you’ll see a table indicating [which packages will be offered to specific Windows 10 device families](#device-family-availability) (and earlier OS versions, if applicable) in ranked order.
+The **Packages** page is where you upload all of the package files (.msix, .msixupload, .msixbundle, .appx, .appxupload, and/or .appxbundle .MSIXVC .XVC (including GameCore)) for the app that you're submitting. You can upload all your packages for the same app on this page, and when a customer downloads your app, the Store will automatically provide each customer with the package that works best for their device. After you upload your packages, you’ll see a table indicating [which packages will be offered to specific Windows 10 device families](#device-family-availability) (and earlier OS versions, if applicable) in ranked order.

```

Comment:
Please move the "and/or" that is currently after ".appxupload" and put it after ".MSIXVC"

Please also add commas between each file extension in the list. 

Then it looks good to go!